api/bag_metadata_import.cpp

```diff
@@ -859,6 +859,7 @@ bool decodeDataQualityInfo(
     else if (schemaVersion == 2)
     {
         //gmd:DQ_DataQuality/gmd:scope/gmd:DQ_Scope/gmd:level

```

Comment:
Would it be better here to change BagDataQuality::scope to be a std::string instead of a char*? It would mean we wouldn't have to use `delete[]` manually everywhere, which might also help with bugs